frontend/packages/console-dynamic-plugin-sdk/src/extensions/dashboards.ts

```diff
@@ -181,32 +223,83 @@ export const isDashboardsOverviewHealthPrometheusSubsystem = (
+): e is DashboardsOverviewHealthURLSubsystem => e.type === 'console.dashboards/overview/health/url';
+
+export const isResolvedDashboardsOverviewHealthURLSubsystem = (
+  e: Extension,
+): e is ResolvedExtension<DashboardsOverviewHealthURLSubsystem> =>
+  e.type === 'console.dashboards/overview/health/url';
 
 export const isDashboardsOverviewHealthResourceSubsystem = (
   e: Extension,
 ): e is DashboardsOverviewHealthResourceSubsystem =>
   e.type === 'console.dashboards/overview/health/resource';
 
+export const isResolvedDashboardsOverviewHealthResourceSubsystem = (
+  e: Extension,
+): e is ResolvedExtension<DashboardsOverviewHealthResourceSubsystem> =>
+  e.type === 'console.dashboards/overview/health/resource';
+
 export const isDashboardsOverviewHealthOperator = (
   e: Extension,
 ): e is DashboardsOverviewHealthOperator =>
   e.type === 'console.dashboards/overview/health/operator';
 
+export const isResolvedDashboardsOverviewHealthOperator = (
+  e: Extension,
+): e is ResolvedExtension<DashboardsOverviewHealthOperator> =>
+  e.type === 'console.dashboards/overview/health/operator';
+
 export const isDashboardsInventoryItemGroup = (e: Extension): e is DashboardsInventoryItemGroup =>
   e.type === 'console.dashboards/inventory/item/group';
 
+export const isDashboardsOverviewUtilizationItem = (
+  e: Extension,
+): e is DashboardsOverviewUtilizationItem => e.type === 'console.dashboards/utilization/item';
+
 export const isDashboardsOverviewInventoryItem = (
   e: Extension,
 ): e is DashboardsOverviewInventoryItem => e.type === 'console.dashboards/inventory/item';
 
+export const isDashboardsOverviewInventoryItemReplacement = (
+  e: Extension,
+): e is DashboardsOverviewInventoryItemReplacement =>
+  e.type === 'console.dashboards/inventory/item/replacement';
+
+export const isProjectDashboardInventoryItem = (e: Extension): e is ProjectDashboardInventoryItem =>
+  e.type === 'console.dashboards/project/item';
+
 export const isDashboardsOverviewResourceActivity = (
   e: Extension,
 ): e is DashboardsOverviewResourceActivity =>
   e.type === 'console.dashboards/overview/activity/resource';
 
+export const isDashboardsOverviewPrometheusActivity = (
+  e: Extension,
+): e is DashboardsOverviewPrometheusActivity =>
+  e.type === 'console.dashboards/overview/prometheus/activity/resource';
+
+export type DashboardsOverviewHealthSubsystem =
+  | DashboardsOverviewHealthURLSubsystem
+  | DashboardsOverviewHealthPrometheusSubsystem
+  | DashboardsOverviewHealthResourceSubsystem
+  | DashboardsOverviewHealthOperator;
+
+export const isDashboardsOverviewHealthSubsystem = (
+  e: Extension,
+): e is DashboardsOverviewHealthSubsystem =>
+  isDashboardsOverviewHealthURLSubsystem(e) ||
+  isDashboardsOverviewHealthPrometheusSubsystem(e) ||
+  isDashboardsOverviewHealthResourceSubsystem(e) ||
+  isDashboardsOverviewHealthOperator(e);
+
 export type DashboardCardSpan = 4 | 6 | 12;
```

Comment:
This file has a lot of extension types and associated type guards.

Futhermore, the amount of support types (e.g. `DashboardCardSpan` and others below) is significant.

I think it would be best to break it down into multiple parts which are easier to manage, for example:
- `dashboards-overview-health.ts`
- `dashboards-overview-inventory.ts`

etc. while putting the support types into a separate file as well, e.g. `dashboards-types.ts`